### PR TITLE
Bump mio

### PR DESCRIPTION
**Pull request by arkpar**
Fixes crashes on windows

### CODE REVIEW

File: util/network/src/lib.rs [hunk 1]
```diff
@@ -54,6 +54,9 @@
 //! }
 //! ```
 
+//TODO: use Poll from mio
+#![allow(deprecated)]
```
Comment (NikVolf) [inline util/network/src/lib.rs]: how about limit this allowance to deprecated mio imports only?
this todo might live for ages there

Comment (arkpar) [inline util/network/src/lib.rs]: Is there a way to do that? Setting an attribute on `extern crate mio` does not work.

Comment (NikVolf) [inline util/network/src/lib.rs]: you have to mark every `use::mio::<DeprecatedItem>` with this `#[allow(deprecated)]` I believe

but let me double-check

Comment (NikVolf) [inline util/network/src/lib.rs]: Ok, this doesn't work, it supresses only `use` itself and still warns when actually used
Disregard this remark, then

**Pull request merged**